bin/merge_describer

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+git log develop.. | egrep '\(#\d+\)$' | while read LINE; do
```

Comment:
I'm not that worried about it since it will mainly be used in the context of the downstreams, but this errors when used on the open branch because `develop` isn't a branch. Just want to mention it.

`fatal: ambiguous argument 'develop': unknown revision or path not in the working tree.`